catalog/core/catalog-core-migratable/src/main/java/org/codice/ddf/catalog/migratable/impl/CatalogMigratableImpl.java

```diff
@@ -0,0 +1,196 @@
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
+import ddf.catalog.filter.FilterBuilder;
+import ddf.catalog.operation.QueryRequest;
+import ddf.catalog.operation.SourceResponse;
+import ddf.catalog.operation.impl.QueryImpl;
+import ddf.catalog.operation.impl.QueryRequestImpl;
+import ddf.catalog.source.CatalogProvider;
+import ddf.catalog.source.UnsupportedQueryException;
+
+/**
+ * Implementation of the {@link org.codice.ddf.migration.Migratable} interface used to migrate
+ * the current catalog provider's {@link Metacard}s.
+ */
+public class CatalogMigratableImpl extends AbstractMigratable {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CatalogMigratableImpl.class);
+
+    private static final String CATALOG_EXPORT_DIRECTORY = "org.codice.ddf.catalog";
+
+    private final CatalogProvider provider;
+
+    private final FilterBuilder filterBuilder;
+
+    private final MigrationFileWriter fileWriter;
+
+    private final CatalogMigratableConfig config;
+
+    private final MigrationTaskManager taskManager;
+
+    private long exportGroupCount;
+
+    /**
+     * Constructor.
+     *
+     * @param description   description of this migratable
+     * @param provider      provider used to retrieve the metacards to export
+     * @param filterBuilder builder used to create query filters
+     * @param fileWriter    object used to write metacards to disk
+     * @param config        export configuration information
+     */
+    public CatalogMigratableImpl(@NotNull String description, @NotNull CatalogProvider provider,
+            @NotNull FilterBuilder filterBuilder, @NotNull MigrationFileWriter fileWriter,
+            @NotNull CatalogMigratableConfig config) {
+        super(description, true);
+
+        notNull(provider, "CatalogProvider cannot be null");
+        notNull(filterBuilder, "FilterBuilder cannot be null");
+        notNull(fileWriter, "FileWriter cannot be null");
+        notNull(config, "Configuration object cannot be null");
+
+        this.provider = provider;
+        this.filterBuilder = filterBuilder;
+        this.fileWriter = fileWriter;
+        this.config = config;
+
+        this.taskManager = createTaskManager(config, createExecutorService(config));
+    }
+
+    /**
+     * Exports all the metacards currently stored in the catalog provider.
+     * <p>
+     * {@inheritDoc}
+     */
+    public MigrationMetadata export(Path exportPath) throws MigrationException {
+        config.setExportPath(exportPath.resolve(CATALOG_EXPORT_DIRECTORY));
+        fileWriter.createExportDirectory(config.getExportPath());
+
+        Collection<MigrationWarning> warnings = new ArrayList<>();
+        Map<String, Serializable> props = new HashMap<>();
+        Filter dumpFilter = filterBuilder.attribute(Metacard.ANY_TEXT)
+                .is()
+                .like()
+                .text("*");
+
+        useNativeQueryMode(props);
```

Comment:
As this is the only call of this method, you could collapse the creation of the map up on line 110 with the addition of the native properties here into a single call. In fact, since no other properties have been added to the map at this point, you can just change your `useNativeQueryMode` method to be a builder method that returns a new map with the single relevant entry.